examples/src/examples/animation/component-properties.tsx

```diff
@@ -196,7 +196,7 @@ class ComponentPropertiesExample {
                     ]
                 };
 
-                const animClipHandler = new pc.AnimClipHandler(app);
+                const animClipHandler = new pc.AnimClipHandler();
```

Comment:
This may be the only reason of the merge conflict, maybe we can just revert and it solves itself?
```suggestion
                const animClipHandler = new pc.AnimClipHandler(app);
```